jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/execution/sam/SamInitRunner.kt

```diff
@@ -0,0 +1,51 @@
+// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.lambda.execution.sam
+
+import com.intellij.execution.configurations.GeneralCommandLine
+import com.intellij.execution.process.CapturingProcessHandler
+import com.intellij.openapi.application.ApplicationManager
+import com.intellij.openapi.vfs.LocalFileSystem
+import com.intellij.openapi.vfs.VfsUtil
+import com.intellij.openapi.vfs.VirtualFile
+import software.amazon.awssdk.services.lambda.model.Runtime
+import software.aws.toolkits.jetbrains.settings.SamSettings
+
+class SamInitRunner {
+    private val samCliExecutable = SamSettings.getInstance().executablePath
+    private var commandLine = GeneralCommandLine(samCliExecutable)
```

Comment:
You should be able to make this a `val` the "wither" returns `this` as a convenience for method chaining - it doesn't actually create a different version of the object so there's no need to do the reassignment.

Reply:
done